srcpkgs/kakoune/template

```diff
@@ -13,8 +13,8 @@ short_desc="Selection-based vim-like editor with less keystrokes"
 maintainer="Jakub Skrzypnik <j.skrzypnik@openmailbox.org>"
 license="Unlicense"
 homepage="https://kakoune.org"
-distfiles="https://github.com/mawww/kakoune/releases/download/v${version}/kakoune-${version}.tar.bz2"
```

Comment:
Why stop using the official tarball? Please explain in commit message.

Reply:
When I created the PR, tarball was unavailable. Reverted this change.